### ISSUE TITLE
icon menu responsiveness in Microsoft Edge

### ISSUE THREAD

**Issue by soniapello** [twbs/bootstrap#24495]
I have a problem with microsoft edge, my menu lost its responsiveness on small menu 
![boo](https://user-images.githubusercontent.com/20027271/31853939-d23a0878-b691-11e7-8314-7a3ba420adf6.png)
the first browser is mozilla and the second on is Microsoft Edge

```
<body id="home">
        <nav class="navbar navbar-expand-sm bg-dark navbar-dark fixed-top">
        	<div class="container">
        		<a href="index.html" class="navbar-brand">Our theme</a>
        		<button class="navbar-toggler" data-toggle="collapse" data-target="#navbarCollapse">
        			<span class="navbar-toggler-icon"></span>
        		</button>
        		<div class="collapse navbar-collapse" id="navbarCollapse">
        			<ul class="navbar-nav ml-auto">
        				<li class="nav-item">
        					<a href="#home" class="nav-link">Home</a>
        				</li>
        				
        				<li class="nav-item">
        					<a href="#explore-head-section" class="nav-link">Explore</a>
        				</li>
        				<li class="nav-item">
        					<a href="#create-head-section" class="nav-link">Create</a>
        				</li>
        				<li class="nav-item">
        					<a href="#share-head-section" class="nav-link">Share</a>
        				</li>
        			</ul>
        		</div>
        	</div>
        </nav>



```

**Comment by XhmikosR** [comment]
Bug reports must include a live demo of the problem. Per our [contributing guidelines](https://github.com/twbs/bootstrap/blob/v4-dev/.github/CONTRIBUTING.md), please create a reduced test case via [CodePen](https://codepen.io/) or [JS Bin](https://jsbin.com/) and report back with your link, Bootstrap version, and specific browser and OS details.

**Comment by soniapello** [comment]
[JS Bin ](https://jsbin.com/sovubavilo/edit?html,output)
bootstrap 4 beta 2
Microsoft Edge 40.15063.674.0 on windows 10

**Comment by XhmikosR** [comment]
Seems to works fine here with Microsoft Edge 41.16299.15.0, which is the current Edge version in Windows 10 Fall Creators update.

**Comment by Ghost180195** [comment]
Works fine on Microsoft Edge 25.10586.0.0
![edge](https://user-images.githubusercontent.com/31181880/31878963-3bf6e4b6-b7f5-11e7-99bf-970ab96c025a.PNG)

**Comment by soniapello** [comment]
The problem was solved with 41.16299.15.0
